packages/web-components/fast-components/src/accordion/accordion-item/accordion-item.ts

```diff
@@ -0,0 +1,34 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+
+export enum AccordionHeadingLevel {
+    h1 = "1",
+    h2 = "2",
+    h3 = "3",
+    h4 = "4",
+    h5 = "5",
+    h6 = "6",
+}
+
+export class AccordionItem extends FASTElement {
+    @attr
+    public headinglevel: AccordionHeadingLevel = AccordionHeadingLevel.h2;
```

Comment:
I think based on previous discussions this should use an attribute of `heading-level` and a property of `headingLevel`